### ISSUE TITLE
Showing an error 

### ISSUE THREAD

**Issue by punitdarji** [wroersma/CTFd-multi-answer#11]
Hi there,
I have installed your plugin and it is showing an error.
CTFd version : 2.3
OS : Windows/Linux
Python : 2.7 and 3.6
![image](https://user-images.githubusercontent.com/38101321/93305601-578f4000-f81c-11ea-9d11-66d5ebbf02bd.png)

**Comment by wroersma** [comment]
I haven't used CTFd myself in over 3 years really so it's not a shock that my code is a little bit broken. I'll try to look into this sometime this week and see if the fix is easy enough.

**Comment by AnykeyNL** [comment]
Got same error here :-( It fails on the first line of the init file on the CTFd.plugins.keys

**Comment by wroersma** [comment]
Yeah, it's a pretty large change, to be honest. I just haven't really had a ton of time recently.

**Comment by AnykeyNL** [comment]
I modified the static answer module to support multi answers. will run my ctf this weekend, if all well I will publish the plugin

**Comment by wroersma** [comment]
What do you mean to publish the plugin? Do you mean to push the changes?

**Comment by jshufelt** [comment]
A link to the mulkti-answers plugin.  https://github.com/AnykeyNL/CTFd-multiFlag-plugin.  It was tested on 3.1.1 and does not have a load() function.

**Comment by wroersma** [comment]
Want this plugin updated? Go and give me one month of support and I'll fix it within 4 hours of the pledge. https://www.patreon.com/vtriple otherwise wait for Anykey to update his "original" plugin
